.github/workflows/run-framework-tests.yaml

```diff
@@ -0,0 +1,53 @@
+on:
+  pull_request:
+    types: [opened, reopened]
+    branches:
+      - main
+
+name: run-framework-tests
+
+jobs:
+  run-framework-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Nest
+        uses: actions/checkout@v4
+        with:
+          path: ''
+          fetch-depth: 0 # Checkout everything to get access to the tags
+          repository: ${{ github.repository }}
+          ref: ${{github.event.pull_request.head.ref}}
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Checkout Flock
+        uses: actions/checkout@v4
+        with:
+          path: 'flock'
+          fetch-depth: 0 # Checkout everything to get access to the tags
+          repository: 'flutter/flutter'
+          ref: 'master'
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      #- name: Apply Nest patches to Flock
+      #  run: |
+      #    tools/git-import-patches patches
+
+      - name: Setup Python
```

Comment:
If you're not going to apply the patches, then do you need python?

Reply:
Well it should apply the patches, but just waiting for there to actually be some patches in the repo to uncomment